src/sentry/api/helpers/group_index.py

```diff
@@ -93,6 +94,10 @@ def build_query_params_from_request(request, organization, projects, environment
         except ValueError:
             raise ParseError(detail="Invalid cursor parameter.")
     query = request.GET.get("query", "is:unresolved").strip()
+    sentry_sdk.set_tag("search.query", query)
```

Comment:
Could the user query contain PII that shouldn't be in our internal data?

Reply:
If it does, it's already stored in `RecentSearches` as well. I'm not really sure what the boundaries are here, do we need to be more careful with error tags?

Reply:
If the text is also in saved searches it is probably fine here too.